src/map.cpp

```diff
@@ -2502,10 +2502,11 @@ ServerMapSector * ServerMap::createSector(v2s16 p2d)
 	/*
 		Do not create over-limit
 	*/
-	if(p2d.X < -MAP_GENERATION_LIMIT / MAP_BLOCKSIZE
```

Comment:
If `MAP_GENERATION_LIMIT` is no longer used anywhere you should remove it to prevent any confusion.